payment-method-basic-card/steps_for_selecting_the_payment_handler.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>
+  Payment Method Basic Card: steps for selecting the payment handler
+</title>
+<link
+  rel="help"
+  href="https://w3c.github.io/payment-method-basic-card/#selecting-the-payment-handler"
+/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
```

Comment:
Please add `setup({ explicit_done: true, explicit_timeout: true });`

Otherwise, at least in Chrome, the test report is printed as soon as the first test is run.